tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -410,9 +462,76 @@ async def test_replicas_tls_key_exchange_command_without_primary(self, bft_netwo
                 nb_fast_path = await bft_network.get_metric(r, bft_network, "Counters", "totalFastPaths")
                 self.assertGreater(nb_fast_path, fast_paths[r])
 
-    async def run_replica_tls_key_exchange_cycle(self, bft_network, replicas):
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    async def test_replicas_tls_key_exchange_command_with_st(self, bft_network):
+        """
+            Operator sends client key exchange command for f - 1 replicas
+        """
+        with log.start_action(action_type="test_replica_key_exchange_command"):
+            bft_network.start_all_replicas()
+            skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+            initial_prim = 0
+            crashed_replica = list(bft_network.random_set_of_replicas(1, {initial_prim}))
+            exchanged_replicas = list(bft_network.random_set_of_replicas(bft_network.config.f - 2, {initial_prim, crashed_replica[0]}))
```

Comment:
here exchanged_replicas list is empty? (because of f-2)

Reply:
Yes, but we later add to it the current primary (tp make sure that the primary key exchange was done by the state transfer handler)